config/initializers/rails_performance.rb

```diff
@@ -0,0 +1,17 @@
+RailsPerformance.setup do |config|
+  config.redis    = Redis::Namespace.new("#{Rails.env}-rails-performance", redis: Redis.new)
+  config.duration = 48.hours
```

Comment:
@ferblape how taxing is going to store longer periods?

Reply:
It depends on the number of requests, but it shouldn't be a problem to store more days, I've updated this to a month and we'll watch the value.